### PR TITLE
fix unit test for different decimal separator for different locales

### PR DESCRIPTION
**Pull request by vitalrev**
👮🏻👮🏻👮🏻 !!!! DESCRIBE YOUR CHANGES HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [x ] Have you run the unit, integration and smoke tests as described [here](https://docs.corda.net/head/testing.html)?
- [x ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/corda/corda/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/corda/corda/blob/master/CONTRIBUTING.md#developer-certificate-of-origin).

Thanks for your code, it's appreciated! :)

The Unit test failed in german location, because the decimal separator is ',' and not '.'
I've changed to the localised decimal separator: DecimalFormatSymbols.getInstance().decimalSeparator

### CODE REVIEW

File: djvm/src/test/kotlin/net/corda/djvm/DJVMTest.kt [hunk 1]
```diff
@@ -4,6 +4,8 @@ import org.assertj.core.api.Assertions.*
 import org.junit.Assert.*
 import org.junit.Test
 import sandbox.java.lang.sandbox
+import java.text.DecimalFormat
```
[the remark this answers is not in the document]
Comment (vitalrev) [reply djvm/src/test/kotlin/net/corda/djvm/DJVMTest.kt]: done

File: webserver/build.gradle [hunk 2]
```diff
@@ -52,6 +52,15 @@ dependencies {
     compile "org.glassfish.jersey.containers:jersey-container-jetty-http:$jersey_version"
     compile "org.glassfish.jersey.media:jersey-media-json-jackson:$jersey_version"
 
+    // Jetty WebSockets
```
Comment (florian-f) [inline webserver/build.gradle]: Please revert this commit, it doesn't belong to this PR.

Comment (vitalrev) [reply webserver/build.gradle]: done

Comment (florian-f) [reply webserver/build.gradle]: nice :+1:
